infrastructure/resource_group_level/modules/gef-ui.bicep

```diff
@@ -0,0 +1,219 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param logAnalyticsWorkspaceId string
+param externalApiHostname string
+param tfmApiHostname string
+param portalApiHostname string
+param redisName string
+
+var dockerImageName = '${containerRegistryName}.azurecr.io/gef-ui:${environment}'
+var dockerRegistryServerUsername = 'tfs${environment}'
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureSettings object = {
+
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+@secure()
+param additionalSecureSettings object = {
+  DOCKER_REGISTRY_SERVER_PASSWORD: 'test-value'
+  DTFS_CENTRAL_API_KEY: 'test-value'
+  EXTERNAL_API_KEY: 'test-value'
+  PORTAL_API_KEY: 'test-value'
+  TFM_API_KEY: 'test-value'
+}
+
+// These values are taken from GitHub secrets injected in the GHA Action
+@secure()
+param secureConnectionStrings object = {
+  COMPANIES_HOUSE_API_KEY : 'test-value'
+  SESSION_SECRET: 'test-value'
+}
+
+// These values are taken from an export of Connection strings on Dev (& validating with staging).
+@secure()
+param additionalSecureConnectionStrings object = {
+}
+
+// TODO:FN-425 use more recent definition?
+resource redis 'Microsoft.Cache/redis@2022-06-01' existing = {
+  name: redisName
+}
+
+var portalApiUrl = 'https://${portalApiHostname}'
+// Note that for externalApiUrl in the CLI script, http was used, but the value in the exported config was https.
+var externalApiUrl = 'https://${externalApiHostname}'
+
+
+// https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
+var azureDnsServerIp = '168.63.129.16'
+
+// These values are hardcoded in the CLI scripts, derived in the script or set from normal env variables
+var settings = {
+  // from env.
+  COMPANIES_HOUSE_API_URL: 'test-value'
+
+  // derived
+  PORTAL_API_URL: portalApiUrl
+  REDIS_HOSTNAME: redis.properties.hostName
+  REDIS_PORT: redis.properties.sslPort
+  // TODO:FN-425 check if this leaks the REDIS_KEY value?
```

Comment:
Note that it was so similar to portal-ui, I just coped it.
The only difference was for some reason `gef-ui` doesn't have `WEBSITE_HEALTHCHECK_MAXPINGFAILURES` set.